packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -253,6 +257,10 @@ render self = vetrinaContainer self $
             , productSelection: self.state.productSelection
             , onLogin: self.props.onLogin
             }
+        ServerError -> self.props.unexpectedError
```

Comment:
Let's only render it in case of `InitializationError` for now, as we are using e.g.`UnexpectedError` everywhere (where we would like to render the other error view instead), and address it when implementing issue #375

Reply:
So for the record should I render the internal `vetrina` error view for cases `ServerError` and `UnexpectedError`?

Reply:
Yes, for now.